src/app/modules/main/module.nim

```diff
@@ -177,7 +177,9 @@ proc createCommunityItem[T](self: Module[T], c: CommunityDto): SectionItem =
     c.isMember,
     c.permissions.access,
     c.permissions.ensOnly,
-    c.members.map(x => member_item.initItem(x.id, x.roles))
+    c.members.map(proc(member: Member): user_item.Item =
```

Comment:
@jrainville I am not sure that I follow why does section item need to know anything about members? They are not used in that context. That item need to have only details needed for the UI, users list for chat section module is available in its users submodule.